importer/main.py

```diff
@@ -486,19 +490,19 @@ def build_payload(resource_type, resources, resource_payload_file):
     for resource in resources:
```

Comment:
This definitely would make the code more readable, however, it might be a bit tricky because the columns in the different csv files are not always the same. For example column 6 is _parentId_ in the locations csv, _organizations_ in the careTeam csv and _alias_ in the organizations csv. We handle the different resourceType [here](https://github.com/onaio/fhir-tooling/blob/ab1b2a4b8106060d4c2c81348800a1d90851b6a7/importer/main.py#L530) within the same for loop. 
Unless we think the readability is worth 4 separate for loops? Is there another way can handle this?

Reply:
nah no worth 4 loops, can you link to the code where that's a problem, just looking through the problem's not clear to me

Reply:
The for loop starts in line [486](https://github.com/onaio/fhir-tooling/blob/ab1b2a4b8106060d4c2c81348800a1d90851b6a7/importer/main.py#L486) then 
- in line [531](https://github.com/onaio/fhir-tooling/blob/ab1b2a4b8106060d4c2c81348800a1d90851b6a7/importer/main.py#L531) we call **organization_extras** in case the resourceType is organizations and pass the resource
- in line [533](https://github.com/onaio/fhir-tooling/blob/ab1b2a4b8106060d4c2c81348800a1d90851b6a7/importer/main.py#L533) we call the **location_extras** in case the resourceType is locations and pass the resource
- in line [535](https://github.com/onaio/fhir-tooling/blob/ab1b2a4b8106060d4c2c81348800a1d90851b6a7/importer/main.py#L535) we call the **care_team_extras** in case the resourceType is careTeams and pass the resource

These three functions then add different objects to the json payload

So if we destructure with the actual names at the beginning of the for loop, we can't pass in the whole resource, and we can't pass the actual names, because they are different for each csv